Objects/fileobject.c

```diff
@@ -762,6 +762,12 @@ file_seek(PyFileObject *f, PyObject *args)
 
     if (f->f_fp == NULL)
         return err_closed();
+    if (f->unlocked_count > 0) {
+        PyErr_SetString(PyExc_IOError,
+            "seek() called during concurrent "
+            "operation on the same file object.");
```

Comment:
I would remove the final ".", it's uncommon in Python error messages.

Reply:
I agree. But this copies the error message at line 431. Do you want to remove the final "." from all error messages in this file?

Reply:
Oh. I looked quickly at fileobject.c error messages, but I didn't notice that the error message already existed in close(). That's a good reason to generalize the check for concurrent operation :-)

In this case, it's fine, keep the final dot :-)